binaries/upload.py

```diff
@@ -63,29 +73,42 @@ def upload_conda_packages():
                     print(f"Anaconda package upload failed for pacakge {name}")
                     return exit_code
 
-    print(f"All packages uploaded to anaconda successfully under the channel {CONDA_USER}")
+    print(
+        f"All packages uploaded to anaconda successfully under the channel {CONDA_USER}"
+    )
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
         description="Upload anaconda and pypi packages for torchserve and torch-model-archiver"
     )
     parser.add_argument(
-        "--upload-conda-packages", action="store_true", required=False, help="Specify whether to upload conda packages"
```

Comment:
should we add a test conda environment as well? One thing I'm not clear on yet is whether it's possible to overwrite binaries in conda in case we make a mistake

Reply:
With Conda, we don't have the same problem as PyPI. We can re-upload with the same name